pandas/tests/io/formats/test_format.py

```diff
@@ -3272,3 +3272,43 @@ def test_filepath_or_buffer_bad_arg_raises(float_frame, method):
     msg = "buf is not a file name and it has no write method"
     with pytest.raises(TypeError, match=msg):
         getattr(float_frame, method)(buf=object())
+
+
+@pytest.mark.parametrize(
+    "inputs, expected",
+    [([" a", " b"], " a\n b"), ([".1", "1"], ".1\n 1"), (["10", "-10"], " 10\n-10")],
+)
+def test_to_string_index_false_corner_case(inputs, expected):
```

Comment:
I think these cases should just be added as param's to your test method below `test_format_remove_leading_space_series`

Reply:
sure, this was just some corner cases that was found during the previous PR. will move.